src/Util/Frame.php

```diff
@@ -49,7 +60,7 @@ public function getFile()
     }
 
     /**
-     * @return int|null
+     * @return integer|null
```

Comment:
@sagikazarmark Not at all. We wouldn't use. "standardclass" though instead of "stdClass" when we document something. See my point?